ambry-api/src/main/java/com/github/ambry/config/RouterConfig.java

```diff
@@ -234,6 +236,20 @@ public class RouterConfig {
   @Default("2")
   public final int routerPutSuccessTarget;
 
+  /**
+   * The maximum number of parallel requests issued at a time by the ReplicateBlob manager.
+   */
+  @Config(ROUTER_REPLICATE_BLOB_REQUEST_PARALLELISM)
+  @Default("3")
+  public final int routerReplicateBlobRequestParallelism;
+
+  /**
+   * The minimum number of successful responses required for a ReplicateBlob operation.
+   */
+  @Config(ROUTER_REPLICATE_BLOB_SUCCESS_TARGET)
+  @Default("2")
```

Comment:
Should we make this 3?
It's likely that we are going to hit the source data node in ReplicateBlob operations, which is a guarantee success (if it's still available). In this case, if we set the success target to 3, then we would probably have one cross-colo replication. This would greatly increase the success of the following TTL_UPDATE operation.

Reply:
About excluding the source host, definitely we can do it. I'll clean up all the ON_DEMAND_TO_DO before production. 

Changing the success number from 2 to 3 may have one side effect.
If we only have two replicas left to replicate the Blob, (may include the source), suppose we can return success status and retry the TtlUpdate successful. But now we think ReplicateBlob failed, so we don't retry the TtlUpdate in the TtlUpdateOperation.
Although client retry may still be successful, I guess it's not our intention.

Reply:
that makes sense, given that we might have to use this feature in a local cluster with any other colos.